index.d.ts

```diff
@@ -98,27 +98,58 @@ export type CommonOptions<EncodingType extends EncodingOption = DefaultEncodingO
 	readonly buffer?: boolean;
 
 	/**
-	Same options as [`stdio`](https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_options_stdio).
-
-	It can also be a file path, a file URL, a web stream ([`ReadableStream`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream)), an [`Iterable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#the_iterable_protocol) or an [`AsyncIterable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#the_async_iterator_and_async_iterable_protocols), unless either [`execaSync()`](#execasyncfile-arguments-options), the [`input` option](#input) or the [`inputFile` option](#inputfile) is used. If the file path is relative, it must start with `.`.
+	[How to setup](https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_options_stdio) the child process' standard input. This can be:
```

Comment:
```suggestion
	[How to setup](https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_process_options_stdio) the child process' standard input. This can be:
```

Should this link point to the LTS docs?

Reply:
Thanks for pointing this out! Fixed.